recipes-browser/cog/cog_0.8.1.bb

```diff
@@ -1,7 +1,7 @@
 require cog.inc
 require conf/include/devupstream.inc
 
-SRC_URI = "https://github.com/Igalia/cog/releases/download/v${PV}/cog-${PV}.tar.xz"
+SRC_URI = "https://github.com/Igalia/cog/releases/download/${PV}/${BP}.tar.xz"
```

Comment:
Could you change this to point be `https://wpewebkit.org/releases/${P}.tar.xz` instead? While at it, I think it would make sense to move the `SRC_URI` variable into `cog.inc` to not have to repeat it in every `.bb` file for each version. The value of `SRC_URI` will always be the same.

Reply:
I agree relying on the wpewebkit.org tarballs is a nice idea. I never really liked hosting these in GitHub either.